cSpell.json

```diff
@@ -6,6 +6,7 @@
     "customizability",
     "Gantt",
     "jison",
+    "mermaid",
```

Comment:
The lint error is for `mermiad's`. 
Isn't mermaid already an english word? Why do we need that in cspell.json?

Reply:
After you clean up cSpell.json in #3753 , I'll update this. (adding "mermaid" into cSpell was my bad. I was kind of blindly correcting "mermiad")

Reply:
@sidharthv96  Actually, looking at the failure, it's not eslint that is failing.  
It's failing in **`docs:verify`**